config/routes.rb

```diff
@@ -149,6 +149,12 @@
     end
   end
 
+  namespace :api do
+    get 'generic_works/:id' => 'generic_works#show'
+    get 'generic_works/:id/add_parent/:parent_id' => 'generic_works#add_parent'
```

Comment:
I think if we implement the front end UI to send a PATCH http request to the resources controller, then we don't need to think about implementing the API controller at all. Is that right? @jcoyne @tpendragon